packages/auth-magic-link-nest/src/magic-link/login.controller.ts

```diff
@@ -29,17 +36,23 @@ export class LoginController {
   async validate(
     @Res({ passthrough: true }) response: Response,
     @ExtractEmailFromMagicLink() email: string,
+    @Query('callbackUrl') callbackUrl: string,
   ) {
     const { frontendUrls } = this.config;
 
     const { userId } = await this.util.findOrCreateByEmail({ email });
+    console.log(callbackUrl);
 
     await this.cookieTokenService.prepareResponseWithNewTokens(
       response,
       userId,
     );
 
-    response.redirect(frontendUrls.auth.redirect.success);
+    if (callbackUrl !== 'default') {
```

Comment:
use the constant for the condition

Reply:
in fact... we should not set the `callbackUrl` if we have the "default" value which in turn leads to the situation where we only have to validate if the `callbackUrl` is present at this stage or not